oauthlib/oauth2/rfc6749/clients/mobile_application.py

```diff
@@ -168,5 +168,5 @@ def parse_request_uri_response(self, uri, state=None, scope=None):
         .. _`Section 3.3`: https://tools.ietf.org/html/rfc6749#section-3.3
         """
         self.token = parse_implicit_response(uri, state=state, scope=scope)
-        self._populate_attributes(self.token)
+        self._populate_token_attributes(self.token)
```

Comment:
Is there test a `populate_attributes` for mobile application?

Reply:
There is now! :+1: